### PR TITLE
add theme provider

### PR DESCRIPTION
**Pull request by kkchu791**

### CODE REVIEW

**Comment by arshiamasih** [comment]
@kkchu791 could you add just a couple bullet points on what exactly you implemented for this PR? this will be super useful for the reviewers and to have documented :) - would you say this issue is complete or has only parts been addressed? thank you!

File: app/src/App.tsx [hunk 1]
```diff
@@ -46,10 +47,11 @@ export const App = () => {
 
   const { isInitializing, isAuthenticated, user } = useAuth0();
 
-  return (
-    <React.Fragment>
-
+  const theme = { mode: 'dark' }
```
Comment (arshiamasih) [inline app/src/App.tsx:50]: seeing the 'dark mode' theme here - is this synonymous to passing in an organization for example? or an additional styling layer?

**Comment by tylerthome** [comment]
Also, can we point this PR to dev instead of master?

**Comment by arshiamasih** [comment]
closing out this PR as we need to look further into potential SC+MUI theming approaches.

**Pull request closed without merging**